### PR TITLE
added parking meters data

### PR DESCRIPTION
**Pull request by jbourak**

### CODE REVIEW

File: R/parking_meters.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#' \describe{
+#'  \item{meter_head}{Meter head type.}
+#'  \item{r_mf_9a_6p}{Rate for Mon-Fri 9AM to 6PM.}
+#'  \item{r_mf_6p_10}{Rate for Mon-Fri 6PM to 10PM.}
+#'  \item{r_sa_9a_6p}{Rate for Sat 9AM to 6PM.}
+#'  \item{r_sa_6p_10}{Rate for Sat 6PM to 10PM.}
+#'  \item{r_su_9a_6p}{Rate for Sun 9AM to 6PM.}
+#'  \item{r_su_6p_10}{Rate for Sun 6PM to 10PM.}
+#'  \item{rate_misc}{Miscellaneous rate information.}
+#'  \item{time_in_effect}{Times that meter is in effect.}
```
Comment (jbourak) [inline R/parking_meters.R:16]: the `time_in_effect` currently has the format `METER IN EFFECT: 9:00 AM TO 10:00 PM`, should I split this into two columns like `in_effect_from` and `in_effect_to`?

Comment (hfboyce) [reply R/parking_meters.R:16]: I think for tidy data, the current column makes the most sense.

File: data-raw/parking_meters.R [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+parking_meters <- raw %>%
+  rename(
+    meter_head = METERHEAD,
+    time_in_effect = TIMEINEFFE,
+    credit_card = CREDITCARD,
+    meter_id = METERID,
+    geo_local_area = `Geo Local Area`
+  ) %>%
+  rename_all(tolower) %>%
+  mutate(geom = str_replace_all(geom, ".*\\[| |\\].*", "")) %>%
```
Comment (hfboyce) [inline data-raw/parking_meters.R:17]: YOU GOT THE REGEX TO WORK THAT I COULDN'T! AMAZING!

**Review by hfboyce — APPROVED** [review]
Thrilled.

**Pull request merged**